examples/Quick_Starts/XV15/Flow360.json

```diff
@@ -0,0 +1,66 @@
+{
+    "geometry" : {
+        "refArea" : 45.604,
+        "momentCenter" : [0, 0, 0],
+        "momentLength" : [3.81, 3.81, 3.81]
+    },
+    "volumeOutput" : {
+        "outputFormat" : "tecplot",
+        "outputFields": ["primitiveVars", "T","Cp", "Mach", "qcriterion", "VelocityRelative"]
+    },
+    "surfaceOutput" : {
+        "outputFormat" : "tecplot",
+	"outputFields": ["primitiveVars", "Cp", "Cf","CfTangent", "CfNormal","nodeForcesPerUnitArea"]
+    },
+    "navierStokesSolver" : {
+        "absoluteTolerance" : 1e-9,
+        "relativeTolerance" : 1e-2,
+        "linearIterations" : 35,
+        "kappaMUSCL" : -1.0,
+        "orderOfAccuracy" : 2,
+        "updateJacobianFrequency" : 4,
+        "equationEvalFrequency" : 1
+    },
+    "turbulenceModelSolver" : {
+        "modelType" : "SpalartAllmaras",
+        "absoluteTolerance" : 1e-8,
+        "relativeTolerance" : 1e-2,
+        "linearIterations" : 25,
+        "kappaMUSCL" : -1.0,
+        "DDES" : true,
+        "orderOfAccuracy" : 2,
+        "updateJacobianFrequency" : 4,
+        "equationEvalFrequency" : 1,
+        "rotationCorrection" : true
+    },
+    "freestream" :
+    {
+        "muRef" : 4.29279e-08,
+        "Mach" : 1.46972e-02,
+        "MachRef" : 0.70,
+        "Temperature" : 288.15,
+        "alphaAngle" : -90.0,
+        "betaAngle" : 0.0
+    },
+    "boundaries" : {
+        "farField/farField" : { "type" : "Freestream" },
+        "innerRotating/blade" :   { "type" : "NoSlipWall" }
+    },
+    "volumeZones":{
+        "innerRotating":{
+            "referenceFrame":{
+                "axisOfRotation" : [0,0,-1],
+                "centerOfRotation" : [0,0,0],
+                "omegaRadians" : 1.84691e-01
```

Comment:
Perhaps show the user how to compute omegaRadians and timeStepSize from RPM and deg/step?